interfaces/kubernetes_manifest/v0/schema.py

```diff
@@ -0,0 +1,56 @@
+"""This file defines the schemas for the provider and requirer sides of the kubernetes_manifest interface.
+
+Examples:
+    RequirerSchema:
+        unit: <empty>
+        kubernetes_manifests: [
```

Comment:
Is this `kuberntes_manifest`, or `app`?  I think this level is talking about whether we're writing in unit or app space.

And then the next level down would be any named attribute inside data